libs/SalesforceSDK/src/com/salesforce/androidsdk/app/SalesforceSDKUpgradeManager.java

```diff
@@ -130,6 +132,9 @@ else if (installedVersion.isGreaterThanOrEqualTo(new SdkVersion(9, 2, 0, false))
             if (installedVersion.isLessThan(new SdkVersion(12, 0, 0, false))) {
                 updateFromBefore12_0_0();
             }
+            if (installedVersion.isLessThan(new SdkVersion(15, 0, 0, false))) {
+                migrateAccountType();
+            }
```

Comment:
I know this isn't the normal naming or upgrade time-fame scheme we typically use, but since this requires the app to implement a value it seemed like a better fit.  Alternatively, we could expose it as a function they can choose to run outside of a MSDK upgrade.